### PR TITLE
[spring-boot-configuration-metadata] Initial integration

### PR DESCRIPTION
**Pull request by kyakdan**

### CODE REVIEW

**Review by Alan32Liu — APPROVED** [review]

File: projects/spring-boot/ConfigurationMetadataRepositoryJsonBuilderFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import org.springframework.boot.configurationmetadata.ConfigurationMetadataRepositoryJsonBuilder;
```
Comment (jonathanmetzman) [inline projects/spring-boot/ConfigurationMetadataRepositoryJsonBuilderFuzzer.java:1]: Can you add a license header please?

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]
See license comment

**Comment by jonathanmetzman** [comment]
Ping

File: projects/spring-boot/ConfigurationMetadataRepositoryJsonBuilderFuzzer.java (hunk 1 above)
Comment (kyakdan) [reply projects/spring-boot/ConfigurationMetadataRepositoryJsonBuilderFuzzer.java:16]: I added a license header and rebased on master

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**